data/json/items/armor.json

```
@@ -10210,5 +10210,112 @@
     "material_thickness": 1,
     "environmental_protection": 1,
     "flags": [ "OVERSIZE", "OUTER", "WATERPROOF", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "bigback",
+    "type": "ARMOR",
+    "name": "high-volume rucksack",
+    "description": "Also know as \"a bigpack\", this rucksack is prized by pack rats but may lead to a bad back.",
+    "weight": 3539,
+    "volume": 85,
+    "price": 20000,
+    "material": [ "steel", "leather" ],
+    "symbol": "[",
+    "color": "black",
+    "covers": [ "TORSO" ],
+    "coverage": 95,
+    "encumbrance": 60,
+    "storage": 400,
+    "warmth": 5,
+    "material_thickness": 6,
+    "flags": [ "BELTED", "WATERPROOF", "ONLY_ONE", "OVERSIZE" ]
+  },
+  {
+    "id": "backpack_hiking",
+    "type": "ARMOR",
+    "name": "hiking backpack",
+    "description": "A large sized hiking backpack with plenty of storage space.",
+    "weight": 2291,
+    "volume": 55,
```

Rounding up a bit.
```suggestion
    "volume": "14 L",
```
No change.
```suggestion
    "volume": "13750 ml",
```